[ko]: add index.md for `web/glossary/placeholder_names` (#16202)
mdn/translated-content: files/ko/glossary/placeholder_names/index.md

---
title: 자리 표시자 이름 (Placeholder names)
slug: Glossary/Placeholder_names
l10n:
  sourceCommit: ada5fa5ef15eadd44b549ecf906423b4a2092f34
---

{{GlossarySidebar}}

> 자리 표시자 이름은 "당사자 A," "도청자" 및 "악의적인 공격자"와 같은 용어를 사용하지 않고, 대화 참가자를 나타내기 위해 암호힉에서 일반적으로 사용됩니다.

Reviewer: ```suggestion
자리 표시자 이름은 "당사자 A(Party A)", "도청자(eavesdropper)" 및 "악의적인 공격자(malicious attacker)"와 같은 용어를 사용하지 않고, 대화 참가자를 나타내기 위해 암호학에서 일반적으로 사용됩니다.
```

"man-in-the-middle", "Party A," "eavesdropper," "malicious attacker." 와같이 따옴표로 감싸진 단어들을 같이 표기하는건 어떤가요?

Author: 넵, 보안 용어들을 좀 더 강조해주는 거 같아 좋아보입니다.

> 자리 표시자 이름은 "당사자 A," "도청자" 및 "악의적인 공격자"와 같은 용어를 사용하지 않고, 대화 참가자를 나타내기 위해 암호힉에서 일반적으로 사용됩니다.

Reviewer: ```suggestion
자리 표시자 이름(placeholder names)은 "당사자 A," "도청자" 및 "악의적인 공격자"와 같은 용어를 사용하지 않고, 대화 참가자를 나타내기 위해 암호힉에서 일반적으로 사용됩니다.
```

다른 문서와 일관되게 추가해도 좋을듯합니다.

가장 일반적으로 사용되는 이름은 아래와 같습니다.

> - 'Alice' 및 'Bob', 서로에게 메시지를 보내고 싶은 두 당사자, 때때로 세 번째 참가자인 'Carol'이 합류합니다.

Reviewer: ```suggestion
- _Alice_ 및 _Bob_, 서로에게 메시지를 보내고 싶은 두 당사자, 때때로 세 번째 참가자인 _Carol_이 합류합니다.
```

영문은 기울임꼴 그대로 사용해도 무방합니다.

> - 'Eve', 앨리스와 밥의 대화를 도청하는 수동적 공격자

Reviewer: ```suggestion
- _Eve_, 앨리스와 밥의 대화를 도청하는 수동적 공격자
```

앨리스와 밥 표기를 영어나 한글로 문서 전체적으로 통일하면 좋을듯합니다.

Author: 이 부분은 굳이 한글로 번역하지 않아도 될 듯해, 영어로 통일하는 방향으로 하겠습니다.

> - 'Mallory', 대화를 수정하고 오래된 메시지를 재생할 수 있는 적극적인 공격자('중간자')

Reviewer: ```suggestion
- _Mallory_, 대화를 수정하고 오래된 메시지를 재생할 수 있는 적극적인 공격자('중간자')
```

> - _Mallory_, 대화를 수정하고 오래된 메시지를 재생할 수 있는 적극적인 공격자('중간자')

Reviewer: ```suggestion
- _Mallory_, 대화를 수정하고 오래된 메시지를 재생할 수 있는 적극적인 공격자(중간자, _man-in-the-middle_)
```
